phase1-coordinator/src/storage/storage.rs

```diff
@@ -31,17 +31,80 @@ impl ContributionLocator {
             is_verified,
         }
     }
+
+    pub fn round_height(&self) -> u64 {
```

Comment:
What is the goal of switching from field access to getters? I can guess that to prevent mutating the structure in other modules. But since you are using this pattern on several structures, it may be helpful to share what problem it addresses.

Reply:
I know a lot of people view it as boilerplate but there are quite a few good reasons to do so https://stackoverflow.com/a/1568230/446250 (minus some of the OO specific points) And also it's consistent with most of the other types in the `aleo-setup` repository which follow the same pattern. In this case, the motivating reason was to ensure that the fields can't be mutated by other modules, but can still be accessed.

Reply:
Do you think it's worth noting this in a documentation comment, or is it obvious enough from the code?